src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -281,12 +286,13 @@ data Parameter proposalMetadata
   -- Admin
   | Set_voting_period VotingPeriod
   | Set_quorum_threshold QuorumThreshold
-  | Flush ()
+  | Flush (Maybe Natural)
```

Comment:
If we hit the operation size limit and have find time dealing with it, we can remove `Maybe` here because the user usually can just pass a large value instead of `Nothing`. Like, if you pass 1000, most likely all proposals will be processed (even if there are more than 1000 in the contract, which is unlikely, processing more than 1000 will likely fail due to gas costs). Anyway, I don't mind having `Maybe` here for now.